### PR TITLE
add ActivePool interface

### PR DESCRIPTION
**Pull request by ruthklirs**

### CODE REVIEW

**Comment by henry-hz** [comment]
check if open zeppelin is compatible with your last cairo

**Comment by ruthklirs** [comment]
I have already added open zeppelin

File: src/interfaces/ItroveManager .cairo [hunk 1]
```diff
@@ -0,0 +1,7 @@
+//
```
Comment (henry-hz) [inline src/interfaces/ItroveManager .cairo:1]: pls fix the file name, and merge the master to your branch and run tests

File: src/interfaces/activePool.cairo [hunk 2]
```diff
@@ -0,0 +1,18 @@
+
+#[starknet::interface]
```
Comment (henry-hz) [inline src/interfaces/activePool.cairo:2]: in interfaces we start with `I` ,    IPool.cairo,    IMycode.cairo, etc...

**Review by henry-hz — CHANGES_REQUESTED** [review]

**Pull request closed without merging**